lib/global.g

```diff
@@ -112,20 +113,66 @@ MAKE_READ_ONLY_GLOBAL("REREADING");
 ##
   
 BIND_GLOBAL := function( name, val)
-    if not REREADING and ISBOUND_GLOBAL( name ) then
+local   pos,  count; 
+    ## ignore if the function is bound and rebinding is permitted 
+    if not ( ISBOUND_GLOBAL( name ) and  
+           ( name in GLOBAL_REBINDING_LIST ) ) then 
+      ## the next 11 lines contain the original code for this function 
+      if not REREADING and ISBOUND_GLOBAL( name ) then
```

Comment:
No need for this comment (this code will be around a long time, this will be the 'original code'). Also these two if statements can be folded into one:

`not REREADING and ISBOUND_GLOBAL(name) and not ( name in GLOBAL_REBINDING_LIST) then` I think